mungegithub/mungers/e2e/e2e.go

```diff
@@ -50,7 +51,8 @@ type BuildInfo struct {
 // RealE2ETester is the object which will get status from a google bucket
 // information about recent jobs
 type RealE2ETester struct {
-	NonBlockingJobNames []string
+	Opts                *options.Options
+	NonBlockingJobNames *[]string
```

Comment:
Why is this a pointer to a slice? Slices are already pointer types.

Does the options module modify slices when reloading, or assign new slices to them?